package.json

```diff
@@ -43,7 +43,7 @@
     "test:ci": "npm run compile && npm run test:lint && npm run test:unit",
     "test:e2e": "npm run compile && cd e2e && wdio run ./wdio.conf.ts",
     "test:lint": "eslint . --ext ts",
-    "test:unit": "vitest --reporter=default --reporter=junit --outputFile=./test-reports/vitest.junit.xml",
+    "test:unit": "vitest --reporter=default --reporter=junit --outputFile=../test-reports/vitest.junit.xml",
```

Comment:
This is kind of confusing because we have `root: 'src'` in the config - I would have expected this to be relative to package.json, but it's relative to the root defined in the config.

I think instead in the config we shouldn't be changing `root`, and we should instead be specifying `dir`: https://vitest.dev/config/#dir
Or just `include`.